client/object_get.go

```diff
@@ -59,23 +59,6 @@ func (x *prmObjectRead) WithinSession(t session.Object) {
 	x.meta.SetSessionToken(&tokv2)
 }
 
-// GetSession returns session object.
-//
-// Returns ErrNoSession err if session wasn't set.
-func (x prmObjectRead) GetSession() (*session.Object, error) {
-	token := x.meta.GetSessionToken()
```

Comment:
But we can deduplicate implementations of course, it's the same thing in each of them.

Reply:
Done